src/pages/lesson-plans/lesson-06.html

```diff
@@ -134,7 +134,7 @@
     </HEAD>
     <BODY>
```

Comment:
This file's purpose was two-fold: 1) to allow me to get feedback on the style and structure changes I've made vs the source .docx file, and 2) to provide something to work with as a starting point for dev that deals with imported lesson plans.

Because of (1), this file is pretty self-contained, including <html>, etc tags and styling.  But in order to use it for (2), I imagine you'll want to strip some of those things out and only use what is currently contained within <body> as a template (plus probably breaking out the scss to its own file).  Rather than change this file (since it is being used for (1) as well), just make new files to use for (2).

Reply:
I've made some educated guesses about the best way to structure things, but by all means if you want to make a different call, do it.  Just communicate back to me somehow about changes you need -- I'll handle the docx import process, so I need to know what you need that's different from what I've provided here.

The way this is currently set up, it assumes you'll find a way to use different template files for the lesson plan based on the selected lesson.  (This file is where you'd get the template for lesson 6.)  But I don't know how hard that might be.  I'm open to suggestions for a different solution.